website/cue/reference/releases/0.20.0.cue

```diff
@@ -0,0 +1,649 @@
+
+				- The addition of `component_discarded_events_total` metric for discarded events
+				- The addition of the `rewritten_timestamp_events_total` metric to count events whose timestamp was rewritten
+				- The log messages for out-of-order events were dropped in severity from `warn` to `debug` since
+				  out-of-order events are a common occurrence, something the sink is designed to handle, and the handling
+				  behavior is explicitly configured by the user
+
+				See the [upgrade guide](highlights/2022-02-08-0-20-0-upgrade-guide#events-loki-sink) for more details.
+				"""
+			pr_numbers: [10971]
+		},
+		{
+			type: "fix"
+			scopes: ["aws_sqs source"]
+			description: """
+				Support for HTTP(S) proxies was added to the `aws_sqs` source. This can be configured the usual way
+				through the [`proxy` configuration field](/docs/reference/configuration/sources/aws_sqs/#proxy) or
+				[environment variables](/docs/reference/configuration/sources/aws_sqs/#https_proxy).
+				"""
+			pr_numbers: [11042]
+		},
+		{
+			type: "fix"
+			scopes: ["vrl"]
+			description: """
+				When a condition (for example for the `route` transform) that was written with VRL fails to execute, the
+				reason for failure is now output in the logs.
+				"""
+			pr_numbers: [11044]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability"]
+			description: """
+				`vector top` has been updated to show the event metrics per output when a component has multiple outputs.
+				"""
+			pr_numbers: [11085]
+		},
+		{
+			type: "enhancement"
+			scopes: ["sources"]
+			description: """
+				TCP-based sources like `fluent`, `syslog`, and `socket` now handle back-pressure better. Previously,
+				back-pressure to these sources from downstream components would cause runaway resource growth as the
+				source would continue to accept connections even though it couldn't handle them right away. A new
+				algorithm has been introduced to start applying back-pressure to clients once ~100k events are buffered
+				in the source. The source will then start closing new connections, rather than accepting them, until it
+				has flushed events downstream.
+				"""
+			pr_numbers: [10962]
+		},
+		{
+			type: "chore"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The minimal supported Kubernetes version was bumped from v1.15 to v1.19.
+
+				Older versions are likely to still work, we just only test against v1.19+.
+				"""
+			pr_numbers: [11118]
+		},
+		{
+			type: "feat"
+			scopes: ["buffers"]
+			description: """
+				This release includes a new opt-in disk buffer implementation that we hope will provide users with
+				faster, more consistent, and lower resource usage buffer performance. We encourage you to opt-in during
+				this beta period and [give us feedback](/community/). See [the beta disk buffer highlight
+				article](/highlights/2022-02-08-disk-buffer-v2-beta) for more details including how to opt-in.
```

Comment:
As a potential improvement to release notes, we could pull out links to highlight articles into a structured element in the object.

We don't (necessarily) need to use it to render them in the description blurb, but it will help us in the future to link highlight articles back to their originating release note and/or linked PRs.